ui/components/app/wallet-overview/non-evm-overview.test.tsx

```diff
@@ -203,7 +203,7 @@ describe('NonEvmOverview', () => {
     const spinner = container.querySelector(
       '.coin-overview__balance .coin-overview__primary-container .spinner',
     );
-    expect(spinner).toBeInTheDocument();
+    expect(spinner).not.toBeInTheDocument();
```

Comment:
When do we show the spinner in this case? if we fallback to 0; and we use isNil to evaluate; does it mean that we never show the spinner?

Reply:
Yes, the spinner will never be shown because we deliberately fall back to 0. It would only appear if we remove that fallback, as you suggested.

Reply:
yes i agree , i just removed the fallback to 0 , let do some tests to make conclusions